CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
 
-### Git setup
+### :point_right: Starting with an issue 
+To get started, find an issue that interests you. If you are new, we recommend selecting a [Good first issue](https://github.com/ecamp/ecamp3/labels/Good%20first%20issue).
+Please note that for other issues we recommend ones with the label [Ready for Implementation](https://github.com/ecamp/ecamp3/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+implementation%22), which signifies that the issue should have clear definitions. If you have any questions, feel free to ask.
+If you are working on an issue, please leave a comment so that we can assign it to you, to make sure that the specifications are still up-to-date, and to prevent two people working on the same issue.
+Alternatively, open a draft pull request and mention the issue ID to signal that you are working on that particular issue.
+Please note that while the wiki can be helpful in understanding the project, it's not exhaustive (meaning there might be parts missing or out of date). If you have any questions, please comment on the issue for clarification. We are happy to help and answer any questions you might have.
+
+### Git setup :octocat:
 
 We use a triangular git workflow. This means that all changes are first pushed to a contributor's fork of the repository, and then the changes are merged into the main fork via a pull request. In practice, setting up this workflow looks as follows:
 
-1. Fork the main repository onto your GitHub account. Let's say your GitHub account is called `your-username`.
+1. Fork the main repository onto your GitHub account. To use the commands your configured git `user.name` must be exactly your git user name. 
+    If you run the code below, and it outputs your GitHub username you are good to go. 
+    ```shell
+    echo $(git config user.name)
+    ```
+    If not you need to replace the `$(git config user.name)` parts with your username or run `git config --global user.name "YourUsername"` with your GitHub username instead of `YourUsername`
+    
 
 2. Clone the main repository to your local computer:
 
-   ```bash
+   ```shell
    git clone https://github.com/ecamp/ecamp3.git
    cd ecamp3
    ```
 
 3. Add your fork as a remote:
 
-   ```bash
-   git remote add your-username https://github.com/your-username/ecamp3.git
+   ```shell
+   git remote add "$(git config user.name)" "https://github.com/$(git config user.name)/ecamp3.git"   
    ```
 
 4. Configure the central repo for pulling the current state and your own repo for pushing new changes:
 
-   ```bash
-   git config remote.pushdefault your-username
+   ```shell
+   git config remote.pushdefault "$(git config user.name)"
    git config push.default current
    ```
 
 Once this is set up, you can start coding, and all `git pull` commands should pull from the central repository by default, while all `git push` commands will push to your fork of the project.
 
-### Starting a new feature
+#### Checkout a feature branch
```

Comment:
In my opinion, this was more beginner-friendly language before. It signified that the instructions above are one-time setup steps, and for every feature after that, you can just start here.

Reply:
I think `branch` is an important keyword here, I was missing in the previous version. In the next section we see `pull request` which is also a new word.

Reply:
I think `branch` is an important keyword here, I was missing in the previous version. In the next section we see `pull request` which is also a new word.